index.html

```diff
@@ -348,12 +348,12 @@
                         <thead><tr><th>Device Button Changes</th><th><code>button</code></th></tr></thead>
                         <tbody>
                             <tr><td>Neither buttons nor touch/pen contact changed since last event</td><td>-1</td></tr>
-                            <tr><td>Left Mouse,<br>Touch Contact,<br>Pen contact (with no modifier buttons pressed)</td><td>0</td></tr>
+                            <tr><td>Left Mouse,<br>Touch Contact,<br>Pen contact</td><td>0</td></tr>
```

Comment:
Actually, just tested this and Wacom Intuos Pro supports hovering left click as an option to be mapped to the two barrel buttons, in addition to the tip. By default the two buttons are set to right click and double click, but it IS possible (and Wacom Intuos Pro not that uncommon a device) 
![wacom-intuos-pro](https://cloud.githubusercontent.com/assets/895831/16398792/0a645516-3cc5-11e6-8336-b820ce6973c9.png)